docs/ascend/ascend.md

```diff
@@ -0,0 +1,70 @@
+# veRL x Ascend
+
```

Comment:
Could you change it to rst format? This way we can display it in the website as well